admin_ui/src/components/NavTranslationsMenu.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <DropDownMenu>
+        <li v-bind:key="key" v-for="(property, key) in $root.$i18n.messages">
+            <a
+                href="#"
+                v-on:click="
+                    $root.$i18n.locale = key
+                    closeDropdown()
+                    getTranslation(key)
+                "
+            >
+                {{ key }}
+            </a>
+        </li>
+    </DropDownMenu>
+</template>
+
+<script lang="ts">
+import Vue from "vue"
+import axios from "axios"
+import DropDownMenu from "./DropDownMenu.vue"
+
+export default Vue.extend({
+    props: {
+        showLanguageDropdown: Boolean
+    },
+    components: {
+        DropDownMenu
+    },
+    methods: {
+        closeDropdown() {
+            this.$emit("closeDropdown", false)
+        },
+        async getTranslation(value) {
+            const response = await axios.get(`./api/languages/${value}`)
```

Comment:
Tiny nit-pick - add a trailing slash to the URL for consistency.

Reply:
`async getTranslation(value: string)` might be better to keep TypeScript happy

Reply:
You're right. I forgot to delete this method. We don't use that. We don't even use the `get_translation` endpoint, but I left it in (I can delete it if you want).